pkg/process/util/address.go

```diff
@@ -74,6 +66,18 @@ func ToLowHigh(addr Address) (l, h uint64) {
 	return
 }
 
+// IPLen returns the size (in bytes) required to represent the address IP
+func IPLen(a Address) int {
```

Comment:
It might be simpler to add this to the `Address` interface as `Len()`, then each implementation is just returning a constant.

Reply:
Sounds good 👍  I wasn't sure if I wanted to extend the interface, but on a second thought `Len()` is probably worth adding